### ISSUE TITLE
Authentication Type not support AAD-federated  sign-in  in MoonCake envrionment

### ISSUE THREAD

**Issue by hundunmuchi** [microsoft/azuredatastudio#8791]
- Azure Data Studio Version:
Version: 1.14.1 (user setup)
Commit: f9b968c1ae514a0d1210788629e1c69125857279
Date: 2019-12-24T02:42:26.303Z
VS Code: 1.38.0
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.17763

Steps to Reproduce:

1. create "New connection";
2. type Azure database server address;
3. select Authentication type , there are only three types can choose:
           a> SQL Login; 
           b> Windows Authentication;
           c> Azure Active Directory - Universal with MFA support
4. then, the "Account" name form field will pop up "Add an account" ,  it will redirect to  device sign-in  page but only for "Microsoft.com" , not included other Azure envrionment.

Would your team please help figure out whether there are any solution or will support it in future?
Apperciate.

Andy

**Comment by Charles-Gagnon** [comment]
@aaomidi - You're adding this as part of your other AAD work, correct?

**Comment by hundunmuchi** [comment]
Thanks a lot for Charles' help and let's waiting for the new big build then. Cheers.

**Comment by aaomidi** [comment]
This is mostly complete, we're waiting on a few internal changes to our application ID before we roll this out.

**Comment by aaomidi** [comment]
US Govt cloud is now enabled. Other environments will soon follow. I'm tracking them in a separate issue: https://github.com/microsoft/azuredatastudio/issues/9832
